Makefile

```diff
@@ -1,4 +1,11 @@
-REGISTRY?=registry.devshift.net
+ifeq ($(TARGET),rhel)
+  DOCKERFILE := Dockerfile.rhel
+  REGISTRY := push.registry.devshift.net/osio-prod
+else
+  DOCKERFILE := Dockerfile
+  REGISTRY := registry.devshift.net
```

Comment:
REGISTRY sould be push.registry.devshift.net

Reply:
nice, catch! thanks